ui-definitions/settings-pages.php

```diff
@@ -451,12 +456,14 @@
 					'type'        => 'panel',
 					'title'       => __( "I'm having an incompatibility issue with a theme, plugin, or hosting environment, what can I do?", 'cloudinary' ),
 					'collapsible' => 'closed',
-					'content'     => sprintf(
-						// translators: The HTML markup.
-						__( 'We’re compatible with most other plugins so we expect it to work absolutely fine. If you do have any issues, please %1$scontact our support team%2$s who will help resolve your issue.', 'cloudinary' ),
-						'<a href="' . Utils::get_support_link() . '" target="_blank" rel="noopener noreferrer">',
-						'</a>'
-					),
+					'content'     => static function () {
+						return sprintf(
+							// translators: The HTML markup.
+							__( 'We’re compatible with most other plugins so we expect it to work absolutely fine. If you do have any issues, please %1$scontact our support team%2$s who will help resolve your issue.', 'cloudinary' ),
+							'<a href="' . Utils::get_support_link() . '" target="_blank" rel="noopener noreferrer">',
```

Comment:
<!-- trivial preference-based requests -->
**🥜 nitpick**
Since there's translated text in the support link, this probably should be escaped.